src/test/scala/com/gu/identity/play/CookieBuilderTest.scala

```diff
@@ -0,0 +1,69 @@
+package com.gu.identity.play
+
+import org.joda.time.format.ISODateTimeFormat
+import org.joda.time.{DateTime, Duration}
+import org.scalactic.Tolerance._
```

Comment:
Ah, ok - so it's adding a bit of tolerance to cope with race conditions - I can live with that.